webvh/webvh/routes.py

```diff
@@ -185,8 +192,9 @@ async def deactivate(request: web.BaseRequest):
 async def witness_get_pending(request: web.BaseRequest):
```

Comment:
Not exactly sure why but there's this `HEAD` option in the api for this endpoint.
![image](https://github.com/user-attachments/assets/2f946e0d-00f7-4c0b-8131-38b2e3168e9d)